models/userinformation.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+
+const UserInformationSchema = new Schema({
+    username: { 
+        type: String, 
+        required : true,
+        unique: true
+    },
+    email: { 
+        type: String, 
+        required : true,
+        unique: true
+    },
+    firstname: { type: String },
+    surname: { type: String },
+    phone: { type: String },
+    address1: { type: String },
+    address2: { type: String },
+    postcode: { type: String },
+    area: { type: String },
+    country: { type: String },
+    state: { type: String },
```

Comment:
Möjligt vi vill lägga till lat/long info som finns i Offer (så vi kan visa vars användaren är på en karta), möjligtvis att vi trycker in det i user istället? då det är oftast den vi hämtar. Osäker!